buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/plugins/lifecycle/LifecyclePlugin.kt

```diff
@@ -184,7 +184,7 @@ class LifecyclePlugin : Plugin<Project> {
             description = "Build production distros, smoke test them and publish"
             group = "publishing"
             dependsOn(":distributions:verifyIsProductionBuildEnvironment", ":distributions:buildDists",
-                ":distributions:integTest", ":docs:check", "publish")
+                ":distributions:integTest", "docs:releaseNotes", "publish")
```

Comment:
```suggestion
                ":distributions:integTest", ":docs:releaseNotes", "publish")
```

I would suggest keeping the colon...

Reply:
Fixed!